app/models/nomis/offender_base.rb

```diff
@@ -1,11 +1,12 @@
 module Nomis
   class OffenderBase
     delegate :home_detention_curfew_eligibility_date,
-             :post_recall_release_date, :post_recall_release_override_date,
+             :post_recall_release_date,
```

Comment:
Is removing this override date intentional? I notice we refer to it in some of the stuff added later?

Reply:
Oh dear. I think I thought I had introduced post_recall_release_override_date from the Swagger Nomis API. When I created the method recall_release_date I remember adding actual_parole_date to the delegate list in OffenderBase without really understanding what the delegate did. And then I realised that I didn't need it there if it was only going to be used in the new method, recall_release_date, in the SentenceDetail class, so I removed it. I also moved it out of attr_accessor to attr_writer in SentenceDetail, as I thought it only needed to be accessible to the SentenceDetail class itself, as it was only being used in my method recall_release_date. I think because post_recall_release_override_date was also part of my recall_release_date method, I did the same for it - I thought that I had introduced it to the code! 
I have just searched post_recall_release_override_date in the code base and it is only being used by me though! That is perhaps what motivated me to remove it from OffenderBase!